cdap-app-fabric/src/test/java/co/cask/cdap/config/UserConfigStoreTest.java

```diff
@@ -281,21 +289,40 @@ public void testAddProfileInProperties() throws Exception {
 
     // this set call should fail since the profile is disabled
     try {
-      prefStore.setProperties("myspace", "app", "workflows", "prog", profileMap);
+      prefStore.setProperties(new ProgramId("myspace", "app", ProgramType.WORKFLOW, "prog"), profileMap);
       Assert.fail();
     } catch (ProfileConflictException e) {
       // expected
     }
     // the pref store should remain unchanged
-    Assert.assertEquals(expected, prefStore.getProperties("myspace", "app", "workflows", "prog"));
+    Assert.assertEquals(expected, prefStore.getProperties(
+      new ProgramId("myspace", "app", ProgramType.WORKFLOW, "prog")));
 
     // enable the profile
     profileStore.enableProfile(profileId);
     expected = profileMap;
-    prefStore.setProperties("myspace", "app", "workflows", "prog", profileMap);
-    Map<String, String> properties = prefStore.getProperties("myspace", "app", "workflows", "prog");
+    prefStore.setProperties(new ProgramId("myspace", "app", ProgramType.WORKFLOW, "prog"), profileMap);
+    Map<String, String> properties = prefStore.getProperties(
+      new ProgramId("myspace", "app", ProgramType.WORKFLOW, "prog"));
     Assert.assertEquals(expected, properties);
 
-    prefStore.deleteProperties("myspace", "app", "workflows", "prog");
+    // test setting at instance level should use profiles in system scope
```

Comment:
I think these tests for setting user profiles at the instance level should be in a separate test method with a descriptive name.